backend/src/main/kotlin/app/ehrenamtskarte/backend/graphql/auth/SignInMutationController.kt

```diff
@@ -6,32 +6,32 @@ import app.ehrenamtskarte.backend.graphql.auth.types.AuthData
 import app.ehrenamtskarte.backend.graphql.auth.types.SignInPayload
 import app.ehrenamtskarte.backend.graphql.exceptions.InvalidCredentialsException
 import com.expediagroup.graphql.generator.annotations.GraphQLDescription
-import jakarta.servlet.http.HttpServletRequest
+import com.expediagroup.graphql.generator.annotations.GraphQLIgnore
 import org.jetbrains.exposed.sql.transactions.transaction
 import org.slf4j.LoggerFactory
 import org.springframework.graphql.data.method.annotation.Argument
+import org.springframework.graphql.data.method.annotation.ContextValue
 import org.springframework.graphql.data.method.annotation.MutationMapping
 import org.springframework.stereotype.Controller
 
 @Controller
-class SignInMutationController(
-    private val request: HttpServletRequest,
-) {
+class SignInMutationController {
+    private val logger = LoggerFactory.getLogger(SignInMutationController::class.java)
```

Comment:
🔧 This could be declared at the file level, since it is not specific to the instance of the containing class. Even better, it could be declared with `by lazy {}`, to speed up the program startup.

Reply:
okay, let's gradually update this everywhere so that we use the same approach in all classes